packages/gestalt/src/Text.js

```diff
@@ -106,6 +108,25 @@ export default function Text({
 }: Props): Node {
   const colorClass = semanticColors.includes(color) && colors[`${color}Text`];
 
+
+
+  let inSemiBoldExp;
+
+  const deviceType = useDeviceType();
```

Comment:
Since you're repeating this logic four times, it could be worth pulling into a temporary custom hook for the duration of the experiment

Reply:
DONE